src/install.rst

```diff
@@ -6,197 +6,314 @@ Installing Nextstrain
-As a result, Mamba can install Conda packages much faster and more accurately than the original Conda installer.
+         a. Docker (recommended) – on Windows, `install Docker Desktop for WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+
+            - Make sure to follow through the last step of enabling **WSL Integration**.
+
+         b. Native – install all the necessary software using mamba:
+
+            .. code-block:: bash
+
+               mamba install -c conda-forge -c bioconda augur auspice snakemake --yes
+
+      7. Confirm that the installation worked.
+
+         .. code-block:: bash
+
+            nextstrain check-setup --set-default
+
+         The final output from the last command should look like this, where ``<option>`` is the option chosen in the previous step:
+
+         .. code-block:: none
+
+            Setting default environment to <option>.
+
+   .. group-tab:: Ubuntu Linux
+
+      .. note::
+
+         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for augur <augur:installation/installation>` and :doc:`install auspice via npm <auspice:introduction/install>`.
+
+      1. Install Miniconda:
+
+         .. code-block:: bash
+
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh
+            # follow through installation prompts
+            rm Miniconda3-latest-Linux-x86_64.sh
+
+      2. Install mamba on the ``base`` conda environment:
+
+         .. code-block:: bash
+
+            conda install -n base -c conda-forge mamba --yes
+            conda activate base
+
+      3. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
+
+         .. code-block:: bash
 
-To try it out, install Mamba.
+            mamba create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
+
+      4. Install the remaining Nextstrain components. There are two options:
+
+         a. Docker (recommended):
+
+            1. Install Docker Engine for Ubuntu using the `convenience script <https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script>`_:
+
+               .. code-block:: bash
+
+                  curl -fsSL https://get.docker.com -o get-docker.sh
+                  sudo sh get-docker.sh
+                  # follow through installation prompts
+                  rm get-docker.sh
+
+            2. Add your user to the `docker` group:
+
+               .. code-block:: bash
+
+                  sudo usermod -aG docker $USER
+
+            3. Restart your machine and activate the conda environment:
+
+               .. code-block:: bash
+
+                  conda activate nextstrain
+
+         b. Native – install all the necessary software using mamba:
+
+            .. code-block:: bash
+
+                  mamba install -c conda-forge -c bioconda augur auspice snakemake --yes
```

Comment:
See comment above about including nextalign and git here.

Reply:
Fixed here: https://github.com/nextstrain/docs.nextstrain.org/pull/81/commits/4584bd292626e4c6671d37e5870157369a7557dd